src/openstudio_lib/OSDocument.cpp

```diff
@@ -1747,6 +1747,7 @@ void OSDocument::openMeasuresBclDlg() {
 }
 
 void OSDocument::on_closeMeasuresBclDlg() {
+  OSAppBase::instance()->checkForRemoteBCLUpdates();
```

Comment:
Wouldn't it be better to check for this in `openMeasuresBclDlg` so you see updated measures when you get there?  Does the update take too long to do there?

Reply:
Oh nice, I see you added a button, that is even better!

Reply:
I have been wondering what the best UX would be, like perhaps when you first go to the scripts tab it tells you whether you have outdated remote BCL measures in your local BCL **and** checks if you have project measures outdated compared to my measures dir, or whether you could it on click of a button.